File: socialuni-server/socialuni-user-sdk/src/main/java/com/socialuni/social/user/sdk/logic/domain/SocialUserFollowDomainInterface.java
package com.socialuni.social.user.sdk.logic.domain;

import com.socialuni.social.user.sdk.model.DO.SocialuniUserFollowDO;
import com.socialuni.social.user.sdk.model.QO.follow.SocialuniUserFollowAddQO;

public interface SocialUserFollowDomainInterface {
    SocialuniUserFollowDO addFlow(SocialuniUserFollowAddQO addVO);

    void cancelFollow(SocialuniUserFollowAddQO addVO);
}

File: socialuni-server/socialuni-follow-sdk/src/main/kotlin/com/socialuni/social/friend/logic/domain/SocialUserFollowDomainInterface.kt
package com.socialuni.social.user.sdk.logic.domain

import com.socialuni.social.friend.dao.DO.SocialuniUserFollowDO
import com.socialuni.social.friend.model.qo.SocialuniUserFollowAddQO

interface SocialUserFollowDomainInterface {
    fun addFlow(addVO: SocialuniUserFollowAddQO?): SocialuniUserFollowDO?

    fun cancelFollow(addVO: SocialuniUserFollowAddQO?)
}
